app/controllers/api/v4/facility_teleconsultations_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V4::FacilityTeleconsultationsController < APIController
+  before_action :set_facility
+  before_action :validate_teleconsultation_facility_belongs_to_users_facility_group
+
+  attr_reader :facility
+
+  def show
+    render json: { teleconsultation_phone_number: teleconsultation_phone_number }
+  end
+
+  private
+
+  def teleconsultation_phone_number
+    "#{facility.teleconsultation_isd_code} #{facility.teleconsultation_phone_number}" if
+      facility.teleconsultation_isd_code && facility.teleconsultation_phone_number && facility.enable_teleconsultation
+  end
+
+  def set_facility
+    @facility ||= Facility.find(params[:facility_id])
+  end
+
+  def validate_teleconsultation_facility_belongs_to_users_facility_group
+    head :unauthorized unless current_facility_group.facilities.where(id: facility.id).present?
```

Comment:
Do we use pundit policies for authorization in APIs? Or is this hand-rolled approach ok?

Reply:
No, we only apply pundit policies to Admin (i.e., Dashboard) users. API user authorization is handled like this. See APIController for reference.